src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -103,6 +106,18 @@ class IqPolicyEvaluatorUtil
     ExceptionUtils.indexOfType(throwable, IOException) >= 0
   }
 
+  private static Properties getAdvancedProperties(final String s, final LoggerBridge loggerBridge) {
+    Properties advanced = new Properties()
+    if (StringUtils.isNotEmpty(s)) {
+      try {
+        advanced.load(new StringReader(s))
+      } catch (IOException e) {
+        loggerBridge.error("Unable to parse advanced properties: ${e.getMessage()}")
```

Comment:
Since it's a StringReader it doesn't necessarily have to be closed but it's pretty jarring to see, could use try with resources or leave as is.

Reply:
Reasonable suggestion, for clarity if nothing else. Fixed.